charts/argo-workflows/values.yaml

```diff
@@ -248,50 +248,48 @@ server:
     enabled: false
     # minAvailable: 1
     # maxUnavailable: 1
-  ## Node selectors and tolerations for server scheduling to nodes with taints
-  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
-  ##
+  # -- Node selectors and tolerations for server scheduling to nodes with taints.
+  # Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+  #
   nodeSelector:
     kubernetes.io/os: linux
   tolerations: []
   affinity: {}
-  # Leverage a PriorityClass to ensure your pods survive resource shortages
+  # -- Leverage a PriorityClass to ensure your pods survive resource shortages
   # ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
   # PriorityClass: system-cluster-critical
   priorityClassName: ""
 
-  # Run the argo server in "secure" mode. Configure this value instead of
+  # -- Run the argo server in "secure" mode. Configure this value instead of
   # "--secure" in extraArgs. See the following documentation for more details
   # on secure mode:
   # https://argoproj.github.io/argo-workflows/tls/
   secure: false
 
-  ## Extra environment variables to provide to the argo-server container
+  # -- Extra environment variables to provide to the argo-server container
   ## extraEnv:
   ##   - name: FOO
   ##     value: "bar"
   extraEnv: []
 
-  # Extra arguments to provide to the Argo server binary.
+  # -- Extra arguments to provide to the Argo server binary.
   extraArgs: []
 
-  ## Additional volumes to the server main container.
+  # -- Additional volumes to the server main container.
   volumeMounts: []
   volumes: []
 
-  ## Ingress configuration.
-  ## ref: https://kubernetes.io/docs/user-guide/ingress/
-  ##
+  # -- Ingress configuration.
+  # ref: https://kubernetes.io/docs/user-guide/ingress/
```

Comment:
Putting the comment here appears to generate a huge default value in the output table, which is hard to parse. Please remove it here and add comments to the individual fields instead.

Reply:
i will add a step in CI to check whether the readme is stale and need to re-generate

Reply:
Hi @tuananh. Thanks for this great contribution. I like this and I also converted several charts to use helm-docs here.
Can you prefix the unnecessary comment line with two `##` so it doesn't get included in the README? This way we can get rid of the long and unnecessary information in the README.

Reply:
@mkilchhofer updated. i also add a step in CI to check if the docs is up to date or not.